src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/ContentPickerConverter.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace Our.Umbraco.Ditto
+{
+    using System;
+    using System.ComponentModel;
+    using System.Globalization;
+
+    using global::Umbraco.Core.Models.PublishedContent;
+    using global::Umbraco.Web;
+
+    /// <summary>
+    /// Provides a unified way of converting content picker properties to strong typed model.
+    /// </summary>
+    /// <typeparam name="T">
+    /// The <see cref="Type"/> of the object to return.
+    /// </typeparam>
+    public class ContentPickerConverter<T> : TypeConverter where T : PublishedContentModel
```

Comment:
Wondering if `PublishedContentModel` constraint is really needed? As it *could* be applied to any POCO?